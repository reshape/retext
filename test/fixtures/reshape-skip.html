<section>
  <p>:tada:</p>
  <p retext-skip>:tada:</p>
</section>
