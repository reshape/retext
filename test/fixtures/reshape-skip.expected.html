<section>
  <p>🎉</p>
  <p>:tada:</p>
</section>
